FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/AppBarLayoutActivity.kt

```diff
@@ -128,16 +128,16 @@ class AppBarLayoutActivity : DemoActivity(), View.OnClickListener {
         updateSearchbar()
     }
 
-    override fun onSaveInstanceState(outState: Bundle?) {
+    override fun onSaveInstanceState(outState: Bundle) {
         super.onSaveInstanceState(outState)
 
         searchbarQuery = searchbar.query.toString()
 
-        outState?.putInt(SCROLL_BEHAVIOR, scrollBehavior.ordinal)
-        outState?.putInt(NAVIGATION_ICON_TYPE, navigationIconType.ordinal)
-        outState?.putBoolean(SEARCHBAR_IS_ACTION_MENU_VIEW, searchbarIsActionMenuView)
-        outState?.putBoolean(SEARCHBAR_HAS_FOCUS, searchbarHasFocus)
-        outState?.putString(SEARCHBAR_QUERY, searchbarQuery)
+        outState.putInt(SCROLL_BEHAVIOR, scrollBehavior.ordinal)
```

Comment:
Any specific reason for this change?

Reply:
Definition of these have changed in androidx , as you can see in the bundle is not nullable now and hence we are overriding correct definition now